index.js

```diff
@@ -233,12 +233,33 @@ var handleCloudWatch = function(event, context) {
   var region = event.Records[0].EventSubscriptionArn.split(":")[3];
   var subject = "AWS CloudWatch Notification";
   var alarmName = message.AlarmName;
-  var metricName = message.Trigger.MetricName;
+  var trigger = message.Trigger;
+  var alarmExpression;
+  if (typeof trigger.MetricName === "undefined") {
```

Comment:
There's also `Array.isArray(trigger.Metrics)` instead of checking `typeof`. While it is a part of the ES5 spec, older browser has spotty implementations. NodeJS should not have a problem with it. If you find that ES6 is acceptable for this file, `Array.isArray` should be acceptable as well.

Reply:
changed to if (typeof trigger.Metrics === 'object')